projects/valtimo/zgw/src/lib/modules/documenten-api/components/documenten-api-tags/documenten-api-tags.component.ts

```diff
@@ -0,0 +1,223 @@
+import {Page} from '@valtimo/document';
+
+@Component({
+  selector: 'valtimo-documenten-api-tags',
+  templateUrl: './documenten-api-tags.component.html',
+  styleUrls: ['./documenten-api-tags.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  standalone: true,
+  imports: [
+    CommonModule,
+    DocumentenApiColumnModalComponent,
+    CarbonListModule,
+    TranslateModule,
+    ConfirmationModalModule,
+    TagModule,
+    ButtonModule,
+    DocumentenApiTagModalComponent,
+    IconModule,
+  ],
+})
+export class DocumentenApiTagsComponent extends PendingChangesComponent implements AfterViewInit {
+  @ViewChild(CarbonListComponent) carbonList: CarbonListComponent;
+
+  private readonly _reload$ = new BehaviorSubject<null | 'noAnimation'>(null);
+  private readonly _documentDefinitionName$: Observable<string> = this.route.params.pipe(
+    map(params => params?.name),
+    filter(docDefName => !!docDefName)
+  );
+
+  public readonly loading$ = new BehaviorSubject<boolean>(true);
+  public readonly fields$ = new BehaviorSubject<ColumnConfig[]>([]);
+  public readonly addModalClosed$ = new BehaviorSubject<boolean>(true);
+  public readonly showDeleteMultipleModal$ = new BehaviorSubject<boolean>(false);
+  public readonly showDeleteModal$ = new Subject<boolean>();
+  public readonly selectedRowKeys$ = new BehaviorSubject<Array<string>>([]);
+  public readonly searchTerm$ = new BehaviorSubject<string>('');
+  public readonly tagToDelete$ = new BehaviorSubject<DocumentenApiTag | null>(null);
+
+  public get documentDefinitionName$(): Observable<string> {
+    return this._documentDefinitionName$;
+  }
+
+  public readonly documentTags$: Observable<DocumentenApiTag[]> = combineLatest([
+    this._documentDefinitionName$,
+    this.searchTerm$,
+    this._reload$,
+  ]).pipe(
+    tap(([_, searchTerm, reload]) => {
+      if (reload === null) {
+        this.loading$.next(true);
+      }
+    }),
+    switchMap(([documentDefinitionName, searchTerm]) =>
+      this.documentenApiTagService.getTagsForAdmin(documentDefinitionName, {
+        page: this.pagination.page - 1,
+        size: this.pagination.size,
+        search: searchTerm,
+      })
+    ),
+    map((tagPage: Page<DocumentenApiTag>) => {
+      this.pagination = {
+        ...this.pagination,
+        collectionSize: tagPage.totalElements,
+      };
+
+      return tagPage.content;
+    }),
+    tap(tags => {
+      this.loading$.next(false);
+    })
+  );
+
+  public readonly ACTION_ITEMS: ActionItem[] = [
+    {
+      label: 'interface.delete',
+      callback: this.openDeleteModal.bind(this),
+      type: 'danger',
+    },
+  ];
+
+  public pagination: Pagination = {
+    collectionSize: 0,
+    page: 1,
+    size: 10,
+  };
+
+  constructor(
+    private readonly route: ActivatedRoute,
+    private readonly documentenApiTagService: DocumentenApiTagService
+  ) {
+    super();
```

Comment:
This is because of the pendingChanges interface, but I don't think it's needed here. Remove the extends PendingChangesComponent

Reply:
Removed